samples/Chat/src/app/utils/utils.ts

```diff
@@ -49,3 +49,18 @@ export const getBackgroundColor = (avatar: string): { backgroundColor: string }
       };
   }
 };
+
```

Comment:
Not a blocker, but I literally just noticed that for all the other apps the file is called `AppUtils` and under chat it is called `utils` we may want to change for consistency.